assets/queries/terraform/aws/rds_database_cluster_not_encrypted/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "656880aa-1388-488f-a6d4-8f73c23149b2",
+  "queryName": "RDS Database Cluster not Encrypted",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "RDS Database Cluster Encryption should be enabled",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_cluster_snapshot#storage_encrypted",
```

Comment:
`https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/db_cluster_snapshot#db_cluster_identifier `

Reply:
@rafaela-soares data sources?

Reply:
My bad. I corrected it